### DISCUSSION TITLE
Variants with design tokens

### DISCUSSION THREAD

**Discussion by KarolusD** [skeletonlabs/skeleton#1500] [Q&A] [answered]
Hi, is there a way to connect variants with design tokens?

Let's say on the light theme I use a button with class `variant-filled-primary`
but on the dark theme this colour is too dim and too dark, I would prefer to use something a bit lighter. (Usually dark themes have less saturated colours)
So something like: ` bg-primary-500-400-token` would be useful but then I have to manually change also the text colour which is a bit inconvenient...

Is there a way to handle this smart way? Something like `variant-filled-primary-500-400-token` ?

**Comment by KarolusD** [comment]
I've solved it using ` bg-primary-500-400-token text-surface-50-900-token` classes but I am wondering if there is something else.

**Reply by Sarenor** [reply, marked as answer]
You can try defining your own custom variant.
